app/controllers/bathrooms_controller.rb

```diff
@@ -24,6 +24,21 @@ def create
     end
   end
 
+  def edit
+    @bathroom = Bathroom.find(params[:id])
+  end
+
+  def update
+    @bathroom = Bathroom.find(params[:id])
+
+    if @bathroom.update(bathroom_params)
+      flash[:notice] = 'Bathroom Edited'
+      redirect_to bathroom_path(@bathroom)
+    else
+      render :edit
```

Comment:
You should use flash to display error messages instead of printing them in the view!